jenkinsapi/fingerprint.py

```diff
@@ -56,8 +56,6 @@ def valid(self):
             # extract the status code from it
             response_obj = err.response
             if response_obj.status_code == 404:
-                logging.warn("MD5 cannot be checked if fingerprints are not "
```

Comment:
This warning was added, AFAIR, on user request: it is explanation on a situation when ``jenkinsapi`` will silently behave not as expected by user when Jenkins is not configured for fingerprints. This message maybe annoying, but it is useful. Please do not remove it.